### DISCUSSION TITLE
Couldn't download app in Japan

### DISCUSSION THREAD

**Discussion by crane-hiromu** [elgatosf/elgato#129] [Help] [answered]
Hi

I'm a fan of Stream Deck.

I work as a software engineer and am trying to connect my stream deck device to my iPad.
However, I can't download the app needed to connect.
-> https://apps.apple.com/app/elgato-stream-deck-connect/id6474433828.

I live in Japan and apparently it's not distributed in Japan.
There is a way to create an overseas Apple ID to download, but please distribute it on the Japanese App Store.

I contacted support and they asked me to post it here, so I did.

Thanks :)

**Comment by r-dent** [comment, marked as answer]
Hi @crane-hiromu !

We had a little delay between releasing the SDK and the app. It should now be available in your region.
